Core/clim-basic/utils.lisp

```diff
@@ -983,36 +905,56 @@ BREAK-STRATEGY may be:
 
 START/END designate the sub-sequence of STRING beginning and ending
 offset. The sub-sequence may contain newline characters and it is up
-to the BREAK-STRATEGY whenever it assigns any meaning to to them."
-  (unless start (setq start 0))
-  (unless end (setq end (length string)))
-  (unless margin (setq margin (* 80 (etypecase width
-                                      (function (funcall width "m" 0 1))
-                                      (number width)))))
-  (unless initial-offset (setq initial-offset 0))
+to the BREAK-STRATEGY whenever it assigns any meaning to to them.
+
+COUNT specifies how many breaks we want to collect."
+  (array-in-bounds-p string start)
+  (array-in-bounds-p string (1- end))
```

Comment:
This doesn't seem to have any effect (besides potential type errors).

Reply:
fixed